### PR TITLE
Add prout tracking

### PR DESCRIPTION
**Pull request by rtyley**
This currently dies on TeamCity - I seem to be having a problem reading the environment variable @SiAdcock !

https://teamcity-aws.gutools.co.uk/viewLog.html?buildId=7331&buildTypeId=memsub_membership_InteractiveBrexitCompanion&tab=buildLog#_

### CODE REVIEW

**Comment by SiAdcock** [comment]
I've set up the Github webhook and post-commit hooks in RiffRaff.

@joelochlann would you mind taking a look at this PR and seeing if it's a 👍 or 👎

File: .prout.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "checkpoints": {
+        "PROD": {
+            "url": "https://interactive.guim.co.uk/2016/05/brexit-companion/embed/embed.html",
+            "overdue": "3H"
```
Comment (joelochlann) [inline .prout.json:5]: seems a bit long?

Comment (joelochlann) [inline .prout.json:5]: oh I guess the idea is we would be manually testing in the meantime... fair enough

**Comment by joelochlann** [comment]
👍

**Pull request merged**